internal/test/mock_clair.go

```diff
@@ -98,7 +98,7 @@ func (c *ClairDouble) postIndexReport(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	reqBodyBuf.WriteByte('\n')
-	err = os.WriteFile(actualPathAbs, reqBodyBuf.Bytes(), 0o666)
+	err = os.WriteFile(actualPathAbs, reqBodyBuf.Bytes(), 0o600)
```

Comment:
This is bad advice. We should use `0o666` and let the user's umask decide which bits to remove.